frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -25,8 +25,15 @@ export const surveysLogic = kea<surveysLogicType>([
         surveys: {
             __default: [] as Survey[],
             loadSurveys: async () => {
-                const response = await api.surveys.list()
-                return response.results
+                const responseSurveys = await api.surveys.list()
+                const surveysResponsesCount = await api.surveys.getResponsesCount()
+
+                responseSurveys.results.forEach((survey) => {
+                    survey.responses_count = 0
+                    survey.responses_count = surveysResponsesCount[survey.id] ?? survey.responses_count
```

Comment:
Minor nit: You don't need the line above, it can be just this

```suggestion
                    survey.responses_count = surveysResponsesCount[survey.id] ?? 0
```